sormas-ui/src/main/java/de/symeda/sormas/ui/samples/environmentsample/EnvironmentSampleGrid.java

```diff
@@ -101,6 +119,19 @@ public EnvironmentSampleGrid(EnvironmentSampleCriteria criteria) {
 			.setRenderer(new DateRenderer(DateFormatHelper.getDateFormat()));
 		((Column<EnvironmentSampleIndexDto, Boolean>) getColumn(EnvironmentSampleIndexDto.RECEIVED)).setRenderer(new BooleanRenderer());
 		((Column<EnvironmentSampleIndexDto, String>) getColumn(EnvironmentSampleIndexDto.LABORATORY)).setMaximumWidth(200);
+		((Column<EnvironmentSampleIndexDto, List<Pathogen>>) getColumn(EnvironmentSampleIndexDto.POSITIVE_PATHOGEN_TESTS))
+			.setRenderer(new TextRenderer() {
+
+				@Override
+				public JsonValue encode(Object value) {
+					if (value != null) {
+
+						return super.encode(
+							String.join(", ", ((List<Pathogen>) value).stream().map(t -> t.getCaption()).collect(Collectors.toList())));
```

Comment:
the value is null conducting to NullPointerException

Reply:
there is the null check above

Reply:
this row still generates a NullPointerException despite the fact that the "value" is not null but it is empty.